config/enabled.yml

```diff
@@ -1279,6 +1279,12 @@ Rails/ReadWriteAttribute:
                  write_attribute(:attr, val).
   Enabled: true
 
+Rails/SafeNavigationOperator:
+  Description: 'Prefer `&.` over `ActiveSupport#try!`.'
```

Comment:
If we want to be perfectly accurate, it's actually `Object#try!`. I guess `ActiveSupport#try!` is a good way of expressing where it comes from, though.